addons/website_links/static/src/js/website_links.js

```diff
@@ -31,6 +31,9 @@ var SelectBox = publicWidget.Widget.extend({
         defs.push(this._rpc({
             model: this.obj,
             method: 'search_read',
+            params: {
+                fields: ['id', 'name']
+            }
```

Comment:
Ideally, add a trailing comma at line 35 and 36 as per our conventions :)

Reply:
@qsm-odoo  Thanks :D